### PR TITLE
Update dependencies + servant-generic + s/Utils//

### PR DESCRIPTION
**Pull request by phadej**

### CODE REVIEW

File: servant-server/src/Servant/Server/Generic.hs [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{-# LANGUAGE DataKinds            #-}
```
Comment (alpmestan) [inline servant-server/src/Servant/Server/Generic.hs]: Should we add the module header with the Copyright thing, to indicate that Patrick wrote this? (and same for the other servant-generic modules)

Comment (phadej) [reply servant-server/src/Servant/Server/Generic.hs]: I added a license note into `Servant.API.Generic`. The only "lines" by Patrick in `Server.Generic` are `Data AsServerT m`, `type AsServerT m :- ..` and `type AsServer`. Not *substantial portion*.

Comment (phadej) [reply servant-server/src/Servant/Server/Generic.hs]: And `Client` is actually more of your and @domenkozar's work.

Comment (alpmestan) [reply servant-server/src/Servant/Server/Generic.hs]: OK, thanks. Looks good to me now.

**Comment by alpmestan** [comment]
Thanks @chpatrick!

**Pull request merged**